data/api/identity/v2_store_invite.yaml

```
@@ -142,20 +142,39 @@ paths:
                   A list of [server's long-term public key, generated ephemeral
                   public key].
                 items:
-                  type: string
+                  type: object
+                  title: PublicKey
+                  properties:
+                    public_key:
+                      type: string
+                      description: |
+                        The public key, encoded using [unpadded Base64](/appendices/#unpadded-base64).
+                    key_validity_url:
+                      type: string
+                      description: |
+                        The URI of an endpoint where the validity of this key can be checked
+                        by passing it as a `public_key` query parameter. See
+                        [key management](/identity-service-api/#key-management).
+                  required: ['public_key', 'key_validity_url']
               display_name:
                 type: string
                 description: The generated (redacted) display_name.
```

Can we fix this while we're here?

```suggestion
                description: The generated (redacted) display name.
```